### DISCUSSION TITLE
Work on non modded clients?

### DISCUSSION THREAD

**Discussion by drakosoldier298** [StoneLabs/working-graves#13] [Q&A] [answered]
Was wondering if this could be put on a server that allows vanilla clients. 
workign on a project thats just optimized server with "modded features" without the need for its players to download mods.
https://gitlab.com/drakosoldier298/choclatey/-/tree/1.19Serv

**Comment by StoneLabs** [comment, marked as answer]
Yes, all my mods are server-side only. Clients can be vanilla.
